### PR TITLE
Support the latest Web Browser for the ODG R-7.

### PR DESCRIPTION
**Pull request by machenmusik**
The changes here use an updated version 0.9.40 of the polyfill that make the accelerometer / gyroscope based sensor fusion work properly on the latest Web Browser for the ODG R-7, which is a landscape-primary device, as well as making A-Frame realize that it should be using it for R-7.

### CODE REVIEW

**Comment by dmarcos** [comment]
Can we open a PR on the polyfill repo?

**Comment by machenmusik** [comment]
ahead of you, https://github.com/googlevr/webvr-polyfill/pull/286

**Comment by machenmusik** [comment]
once @jsantell merges into polyfill, if we are using latest master of polyfill in general, then the package.json change can be reverted

**Comment by dmarcos** [comment]
Good! I prefer to wait for googlevr/webvr-polyfill#286 to land and just bump the `webvr-polyfill`. Cleaner and less work.

**Comment by dmarcos** [comment]
Pointing dependencies to private of forks should be just a last resource

**Comment by machenmusik** [comment]
understood.  looks like it just merged, let me retry without package.json, hang tight

**Comment by jsantell** [comment]
just published webvr-polyfill v0.9.40 with the changes to support the R7 :metal:

**Comment by machenmusik** [comment]
nice!  OK so now the latest ODG R-7 Web Browser can use polyfill with correct accelerometer/gyro behavior, so you can load up https://r7-aframe.glitch.me/ and look at the boxes to select them.  (Although the display is still monoscopic at the moment.)

**Comment by machenmusik** [comment]
@dmarcos OK to merge, thx @jsantell !

File: package.json [hunk 1]
```diff
@@ -32,9 +32,9 @@
   "dependencies": {
     "@tweenjs/tween.js": "^16.8.0",
     "browserify-css": "^0.8.2",
-    "debug": "ngokevin/debug#noTimestamp",
+    "debug": "github:ngokevin/debug#noTimestamp",
```
Comment (dmarcos) [inline package.json]: Why is this necessary?

File: package.json [hunk 2]
```diff
@@ -32,9 +32,9 @@
...
     "deep-assign": "^2.0.0",
-    "document-register-element": "dmarcos/document-register-element#8ccc532b7",
+    "document-register-element": "github:dmarcos/document-register-element#8ccc532b7",
```
Comment (dmarcos) [inline package.json]: Why is this change necessary?

**Comment by machenmusik** [comment]
those github: changes shouldn't have been necessary, I undid by checking out an older version, let me fix

**Comment by machenmusik** [comment]
ok, package.json fixed

**Comment by machenmusik** [comment]
looks correct now:
- bump polyfill to 0.9.40, which has R-7 changes
- make A-Frame use accelerometer and gyro for R-7 (with polyfill fixes), like mobile phones
- tweak landscape utils call to work for R-7

**Comment by dmarcos** [comment]
Thanks! 🏅
